classes/output/tabs.php

```diff
@@ -60,6 +60,12 @@ public function export_for_template(\renderer_base $output): array {
                     'title' => get_string('report_slowest_other', 'tool_excimer'),
                     'text' => get_string('tab_slowest_other', 'tool_excimer'),
                 ],
+                [
+                    'id' => 'course',
+                    'link' => [['link' => new \moodle_url('/admin/tool/excimer/slow_course.php')]],
```

Comment:
Why is the link a double array? Im looking at the wider pattern and it seems really bizarre

Reply:
Not sure it seems redundant, since at the bottom it just calls `link[0]->link`

https://github.com/catalyst/moodle-tool_excimer/blob/5257c205ba836337b01b9f5f2780efea73332bcc/classes/output/tabs.php#L90-L94

Reply:
Was added in 5a1bad9 - maybe the idea was multiple links per tab?